Header header
time timestamp
sensor_msgs/CompressedImage img
detection_osnet/ProcessWindow[] data